src/workloads/scale/MixedWorkloadsGenny.yml

```diff
@@ -0,0 +1,354 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload is a port of the mixed_workloads in the workloads
+  repo. https://github.com/10gen/workloads/blob/master/workloads/mix.js. It runs 4 sets of
+  operations, each with dedicated actors/threads. The 4 operations are insert, findOne, updateOne,
+  and deleteOne. Since each type of operation runs in a dedicated thread it enables interesting
+  behavior, such as reads getting faster because of a write regression, or reads being starved by
+  writes. The origin of the test was as a reproduction for BF-2385 in which reads were starved out
+  by writes.
+
+# This workload does not support sharding yet.
+
+
+# These two values should match those are the top of MixPhases.yml
+dbname: &dbname mix
+DocumentCount: &NumDocs 100000
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2000
+
+
+Actors:
+- Name: Setup
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    BatchSize: 100
+    Threads: 1
+    DocumentCount: *NumDocs
+    Database: *dbname
+    CollectionCount: 1
+    Document: &doc
+      id: {^RandomInt: {min: 0, max: *NumDocs}}
+      a: {^RandomInt: {min: 0, max: 1000000}}
+      # Note that in the original workload the string c was perfectly compressable. We can put a
+      # constant there if needed.
+      c: &string {^RandomString: {length: 50}}  # Adjust this so the doc comes out as 100 B.
+    Indexes:
+    - keys: {id: 1}
+    - keys: {a: 1}
+  - Phase: 1..10
+    Nop: true
+
+- Name: QuiesceBetweenLevels
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - &nop {Nop: true}
+  - &quiesce
+    Repeat: 1
+    Database: admin
+    Operations:
+    # Fsync to force a checkpoint and quiesce the system.
+    - OperationMetricsName: FsyncCommand
+      OperationName: AdminCommand
+      OperationCommand:
+        fsync: 1
+  - *nop
+  - *quiesce
+  - *nop
+  - *quiesce
+  - *nop
+  - *quiesce
+  - *nop
+  - *quiesce
+  - *nop
+
+- Name: Update_1
+  Type: CrudActor
+  Database: *dbname
+  Threads: 1
+  Phases:
+  - Phase: 0..1
+    Nop: true
+  - Threads: 1
```

Comment:
Do I need this threads entry? I don't remember anymore. I have it set on the actor.

Reply:
I don't believe this is necessary. Your call if you want to do another patch-test with it removed - probably worth it given how many times the per-phase Threads is used in this doc

Reply:
I'll take them out but verify it still works correctly. They may have been added in out of superstition. I don't recall.